desktop/src/main/java/bisq/desktop/main/dao/governance/result/VoteResultView.java

```diff
@@ -157,9 +165,11 @@ public VoteResultView(DaoFacade daoFacade,
     @Override
     public void initialize() {
         gridRow = phasesView.addGroup(root, gridRow);
+
         selectedVoteResultListItemListener = (observable, oldValue, newValue) -> onResultsListItemSelected(newValue);
 
         createCyclesTable();
+        exportButton.updateText(Res.get("shared.exportJSON"));
```

Comment:
Here would be the code or creating the Button. 
E.g. `button = FormBuilder.addButtonAfterGroup( gridPane,  rowIndex,  title);`